install/kubernetes/helm/subcharts/tracing/values.yaml

```diff
@@ -8,6 +8,9 @@ jaeger:
   tag: 1.7
   memory:
     max_traces: 50000
+  resources: {}
```

Comment:
I'm a little confused why `configFileEnabled` and `secretFileEnabled ` are under `resources ` section. 
IMHO, this will break resources settings for Jaeger deployment, See:https://github.com/istio/istio/blob/master/install/kubernetes/helm/subcharts/tracing/templates/deployment.yaml#L67-L72